pkg/apis/apps/install/install.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Not a fan of `apps`, since lots of apps won't be pet sets.  `clustered` is too close to the `cluster` resource.